creates/sendSMSMessage.ts

```diff
@@ -10,16 +10,16 @@ const sendSMSMessageInput = [
   {
     key: "workspaceId",
     required: true,
-    label: "workspaceId",
+    label: "Workspace ID",
     helpText: "The identifier for the workspace.",
     dynamic: "workspaces.id.projectName",
   },
   {
     key: "to",
     required: true,
-    label: "To",
+    label: "Mobile Phone Number", 
     helpText:
-      "Allows a combination of phone numbers, email address, Whispir Contact MRIs, User MRI, Distribution List MRI, separated by ,",
+      "Allows a com   bination of phone numbers, email address, Whispir Contact MRIs, User MRI, Distribution List MRI, separated by ,",
```

Comment:
@whutomo  - please remove extra space here and replace separated by to `separated by comma (,)`